Project.toml

```
@@ -108,7 +108,7 @@ StructArrays = "0.6.18, 0.7"
 SummationByPartsOperators = "0.5.52"
 T8code = "0.7.4"
 TimerOutputs = "0.5.23"
-Triangulate = "2.2"
+Triangulate = "2.2, 3"
```

```suggestion
Triangulate = "3"
```
To see, which package hinders the update to v3.